manifests/charts/gateways/istio-ingress/templates/deployment.yaml

```
@@ -264,6 +268,10 @@ spec:
 {{ toYaml $gateway.additionalContainers | indent 8 }}
 {{- end }}
       volumes:
+      - emptyDir:
```

```suggestion
      - emptyDir: {}
```

This is probably super nitpicky but its consistent with below and I have a very small worry that somehow something will treat them differently

---

I updated all the charts with that suggestion. Thanks